packages/envd/internal/events/events.go

```diff
@@ -0,0 +1,59 @@
+package events
```

Comment:
Why is this file part of a "typo" commit (https://github.com/e2b-dev/infra/pull/1107/commits/fa6d9830d126264b983daa1af892b987b69b6a65)? I suppose it belongs to future implementation, revert it now.

Reply:
fixed